manifests/resource/location.pp

```diff
@@ -212,6 +214,8 @@
   Boolean $mp4                                        = false,
   Boolean $flv                                        = false,
   Optional[String] $expires                           = undef,
+  Optional[Enum['on','off']] $log_not_found           = undef,
```

Comment:
Should probably be `Optional[Variant[String, Array]]`
https://github.com/voxpupuli/puppet-nginx/blob/master/manifests/resource/server.pp#L101-L106
https://github.com/voxpupuli/puppet-nginx/blob/master/templates/server/server_header.erb#L139-L161